pkgdiff.pl

```diff
@@ -3153,7 +3185,7 @@ ($$)
         if(unpackArchive(abs_path($Path), $CPath)!=0) {
             exitStatus("Error", "can't extract package \'".getFilename($Path)."\'");
         }
-        
```

Comment:
Please revert indentation space back (and squash commits) to make patch readable.

Reply:
Ok :+1: